config/routes.rb

```diff
@@ -72,89 +72,90 @@
       post '/carry-over' => 'carry_over#create', as: :carry_over
 
       scope '/personal-details' do
-        get '/' => 'personal_details/base#new', as: :personal_details_new
-        post '/' => 'personal_details/base#create', as: :personal_details_create
-        get '/edit' => 'personal_details/base#edit', as: :personal_details_edit
-        post '/edit' => 'personal_details/base#update', as: :personal_details_update
+        get '/' => 'personal_details/base#new', as: :personal_details
+        post '/' => 'personal_details/base#create'
+        get '/edit' => 'personal_details/base#edit', as: :edit_personal_details
+        patch '/edit' => 'personal_details/base#update'
         get '/nationalities' => 'personal_details/nationalities#new', as: :nationalities
         post '/nationalities' => 'personal_details/nationalities#create'
         get '/nationalities/edit' => 'personal_details/nationalities#edit', as: :edit_nationalities
-        post '/nationalities/edit' => 'personal_details/nationalities#update'
+        patch '/nationalities/edit' => 'personal_details/nationalities#update'
         get '/languages' => 'personal_details/languages#edit', as: :languages
         post '/languages' => 'personal_details/languages#update'
         get '/languages/edit' => 'personal_details/languages#edit', as: :edit_languages
-        post '/languages/edit' => 'personal_details/languages#update'
+        patch '/languages/edit' => 'personal_details/languages#update'
```

Comment:
Can line 84 be removed? I removed locally ran the specs and they all passed.

Reply:
I should have changed those to new/create. I feel like using new/create edit/update is probably a better pattern to follow. Although the functionality in the actions is near identical so it might be overkill.